templates/Web/Pages/Angular.Grid/src/app/app-shell/Param_SourceName_Kebab/grid-box/grid-box.component.spec.ts

```diff
@@ -1,4 +1,5 @@
 ﻿import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { MatIconModule } from '@angular/material/icon';
```

Comment:
Hmm do we need material's icon module? 
Refer to: #766
There are other uses of this in this PR as well.

Reply:
We are going to remove all material design from generated code. Thus, we are going to remove all material design code ( including this one) from generated code and other styling change in another PR #766.